gather.go

```diff
@@ -29,13 +29,33 @@ type Data struct {
 		Labels      map[string]string `json:"labels"`
 		ClusterId   string            `json:"clusterId"`
 		NodeName    string            `json:"nodeName"`
+		// HostInfo for hosts
+		HostInfo *HostInfo `json:"info"`
 		// LaunchConfig for services
 		LaunchConfig *LaunchConfig `json:"launchConfig"`
 		// ComponentStatuses for clusters
 		ComponentStatuses []*ComponentStatuses `json:"componentStatuses"`
 	} `json:"data"`
 }
 
+type HostInfo struct {
+	CPUInfo struct {
+		Count int `json:"count"`
+	} `json:"cpuInfo"`
+	MemoryInfo struct {
+		MemTotal int `json:"memTotal"`
+		MemFree  int `json:memFree`
```

Comment:
@astronaut808 Can you add quotes here?
+ MountPoint.Used

Reply:
@leominov Thank you, I fixed it.